src_new/creative/owCreativeRenderer.js

```diff
@@ -0,0 +1,44 @@
+require("prebid-universal-creative");
+
+window.PWT = window.PWT || {};
+window.pbjs = window.pbjs || {};
+
+var refThis = this;
+
+// Desc: Uses prebid js render ad function to call cache and render ad based on the params from response.
+// Param Defination 
+exports.renderCreative = function (cacheUrl, cachePath, uuid) {
+	if (cacheUrl && uuid) {
+		window.pbjs.renderAd(document, "", {
+			cacheHost: cacheUrl,
+			cachePath: cachePath,
+			uuid: uuid,
+			mediaType: "banner",
+		});
+	}
+};
+
+exports.removeProtocolFromUrl = function(url){
+	var outputUrl = url || "";
+	if(url && url.length > 0){
+		outputUrl = url.replace(/^http:\/\//i, "");
+		outputUrl = url.replace(/^https:\/\//i, "");
+	}
+	return outputUrl;
+}
+
+///  Change name to general render function : renderOWCreative
+window.PWT.renderOWCreative = function (targetingKeys) {
+	if (targetingKeys) {
+		var cacheid = targetingKeys.cacheid || "";
+		var cacheURL = targetingKeys.cacheURL || "";
+		var cachePath = targetingKeys.cachePath || "/cache";
+		if (cacheURL.length > 0 && cacheid.length > 0) {
+		    cacheURL = refThis.removeProtocolFromUrl(cacheURL); // removes protocol from url if present and returns host only
+			refThis.renderCreative(cacheURL, cachePath, cacheid);
```

Comment:
pass an object as an argument to be able to pass more information in future

Reply:
Done